File: String/Coin-Toss-Score/CoinTossScore.js
function coinTossScore(tosses) {
  let result = 0
  for (let i = 0; i < tosses.length; i++) {
    if (tosses[i] === 'H') {
      result += 2
    } else {
      result -= 1
    }
  }
  console.log('Points:', result)
}

// Example usage
coinTossScore('HTHHTTHTHHH')

File: String/Coin-Toss-Score/coin_toss_score.py
def coin_toss_score(tosses):
    result = 0
    for ch in tosses:
        if ch == 'H':
            result += 2
        else:
            result -= 1
    print("Points:", result)

# Example usage
coin_toss_score("HTHHTTHTHHH")
